src/aten/sycl/ActivationOpsKernels.cpp

```diff
@@ -0,0 +1,200 @@
+#include <ATen/ATen.h>
+#include <ATen/Dispatch.h>
+#include <ATen/NumericUtils.h>
+#include <ATen/native/Activation.h>
+#include <ATen/native/TensorIterator.h>
+#include <aten/sycl/Loops.h>
+
+namespace at {
+namespace native {
+namespace xpu {
+
+template <typename scalar_t>
+struct ReluFunctor {
+  scalar_t operator()(scalar_t x) const {
+    return x <= scalar_t{0} ? scalar_t{0} : x;
+  }
+};
+
+template <typename scalar_t>
+struct ThresholdFunctor {
+  scalar_t operator()(scalar_t x, scalar_t other) const {
+    return x <= threshold_ ? value_ : other;
+  }
+
+  ThresholdFunctor(scalar_t threshold, scalar_t value)
+      : threshold_(threshold), value_(value) {}
+
+ private:
+  scalar_t threshold_;
+  scalar_t value_;
+};
+
+// constexpr float M_SQRT2 = 1.41421356237309504880;
+// constexpr float M_2_SQRTPI = 1.12837916709551257390;
+// constexpr float M_SQRT1_2 = 0.70710678118654752440;
+
+template <typename scalar_t>
+struct GeluTanhFunctor {
+  scalar_t operator()(scalar_t x) const {
+    using opmath_t = at::opmath_type<scalar_t>;
+    constexpr opmath_t kBeta = M_SQRT2 * M_2_SQRTPI * opmath_t(0.5);
+    constexpr opmath_t kKappa = 0.044715;
+    auto x_cube = static_cast<opmath_t>(x) * static_cast<opmath_t>(x) *
+        static_cast<opmath_t>(x);
+    auto inner = kBeta * (static_cast<opmath_t>(x) + kKappa * x_cube);
+    return opmath_t(0.5) * static_cast<opmath_t>(x) *
+        (opmath_t(1) + ::tanh(inner));
```

Comment:
what's the implementation?
<img width="708" alt="image" src="https://github.com/intel/torch-xpu-ops/assets/28250760/24d0ec39-567f-432e-8b27-707d314a1008">